packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/ios/initializeflow.md

```diff
@@ -0,0 +1,45 @@
+You can provide the configuration information in a property list file or in the initialization call.
+
+#### Add configuration with a property list
+
+The default `init()` function of `InteractionCodeFlow` looks for a property list called `Okta.plist`. You can also call `init(issuer:clientId:scopes:redirectUri:additionalParameters:)` to set the configuration values.
+
+To use a property list, first create a property list file, such as `Okta.plist`. Next, add key-value pairs with the configuration settings for your application integration. For example, in the following text version of a `plist`, substitute your issuer URL for `{yourIssuerUrl}`, the client ID of your application integration for `{yourClientId}`, and a URI that launches your app. Set the scopes based on the access required by your app.
+
+```properties
```

Comment:
This should still be xml as it's not actually a .properties file.
```suggestion
```xml
```

Reply:
Changed it back in the next commit